.github/workflows/release.yaml

```diff
@@ -27,23 +27,22 @@ jobs:
         with:
           version: v3.8.1
 
-      # Optional step if GPG signing is used
-#      - name: Prepare GPG key
-#        run: |
-#          gpg_dir=.cr-gpg
-#          mkdir "$gpg_dir"
-#
-#          keyring="$gpg_dir/secring.gpg"
-#          base64 -d <<< "$GPG_KEYRING_BASE64" > "$keyring"
-#
-#          passphrase_file="$gpg_dir/passphrase"
-#          echo "$GPG_PASSPHRASE" > "$passphrase_file"
-#
-#          echo "CR_PASSPHRASE_FILE=$passphrase_file" >> "$GITHUB_ENV"
-#          echo "CR_KEYRING=$keyring" >> "$GITHUB_ENV"
-#        env:
-#          GPG_KEYRING_BASE64: "${{ secrets.GPG_KEYRING_BASE64 }}"
-#          GPG_PASSPHRASE: "${{ secrets.GPG_PASSPHRASE }}"
+      - name: Prepare GPG key
+        run: |
+          gpg_dir=.cr-gpg
+          mkdir "$gpg_dir"
+
+          keyring="$gpg_dir/secring.gpg"
+          base64 -d <<< "$GPG_HELM_SIGNING_KEY_BASE64" > "$keyring"
+
+          passphrase_file="$gpg_dir/passphrase"
+          echo "$GPG_HELM_SIGNING_PASS" > "$passphrase_file"
+
+          echo "CR_PASSPHRASE_FILE=$passphrase_file" >> "$GITHUB_ENV"
+          echo "CR_KEYRING=$keyring" >> "$GITHUB_ENV"
+        env:
+          GPG_KEYRING_BASE64: "${{ secrets.GPG_KEYRING_BASE64 }}"
```

Comment:
These need toatch the secrets naming (ie the github secrets we created)

Reply:
Switched the env variables to the right side.